### ISSUE TITLE
Encountered a bug

### ISSUE THREAD

**Issue by imposd** [tarkah/grout#6]
I'm unsure on how to re-create this issue, but I can tell you the steps to maybe reproduce this.

After pushing CRTL + ALT + S a few times the window with the [grids](https://awau.moe/9oSXLNC.png) went [white](https://awau.moe/9C6cfJQ.png) and I was unable to put anything in that area, as if it were still on my screen, but not working, and in programs like discord it kind of just blended inside of the program, not allowing me to click in that area. And re pushing CRTL + ALT + S did nothing, so I had to force close the program

**Comment by tarkah** [comment]
Thanks for the bug report! I was facing the same issue yesterday. I'm going to need to litter some trace logging throughout the program to help catch this and other bugs.

**Comment by tarkah** [comment]
@dvhe I figured this one out. Apparently one of the resources I was using during drawing the UI doesn't get freed up automatically. So a memory leak started, which then freezed everything up. This was very easy to trigger with a very large grid.

There is a function to free it up that I added and now it no longer freezes! I'll publish a new release sometime today, I'm going to fold in a few more feature updates first.
